eng/scripts/DownloadDevopsLogger.ps1

```diff
@@ -0,0 +1,6 @@
+New-Item -ItemType directory artifacts -Force
+Invoke-WebRequest -Uri https://vstsagenttools.blob.core.windows.net/tools/msbuildlogger/5/msbuildlogger.zip -OutFile artifacts/msbuildlogger.zip
```

Comment:
We probably should add some retries here if we stick to downloading it ourselves.